src/main/java/org/mule/module/apikit/metadata/internal/amf/FlowMetadata.java

```diff
@@ -243,9 +240,16 @@ private ObjectTypeBuilder getQueryParameters(final Operation operation) {
 
   private void addFields(ObjectTypeBuilder builder, EndPoint endPoint) {
     List<Parameter> parameters = endPoint.parameters();
+    Consumer<Parameter> parameterConsumer = p -> builder.addField().key(p.name().value()).value(metadata(p))
+        .required(p.required().value());
     if (!parameters.isEmpty()) {
-      parameters.forEach(p -> builder.addField().key(p.name().value()).value(metadata(p))
-          .required(p.required().value()));
+      parameters.forEach(parameterConsumer);
+    } else {
```

Comment:
can you add context for this?

Reply:
Javadoc added for clarifying. Please let me know if it is still not clear enough.